test/run_test

```diff
@@ -29,6 +33,7 @@ test -n "${OS-}" || false 'make sure $OS is defined'
 
 CIDFILE_DIR=$(mktemp --suffix=postgresql_test_cidfiles -d)
 
+volumes_to_clean=
```

Comment:
This variable is not modified nowhere

Reply:
Above mentioned problem is fixed now.
---------------------------------------------
Technically this is not needed - `ct_path_append` creates a variable if it doesn't exist...